docs/tutorials/tutorial2.md

```diff
@@ -280,13 +280,13 @@ Jobs:
     [✓] samtoolsflagstat (N/A)       
 
 Outputs:
-    - stats: $HOME/janis-tutorials/tutorial2/stats.txt
+    - stats: $HOME/janis-tutorials/tutorial2/stats
```

Comment:
There was no `stats.txt` for me, only `stats`? Not sure if it was supposed to be `.txt`, maybe a different version of the tool used, or maybe I missed a step somewhere?